charts/infra-server/templates/server/deployment.yaml

```diff
@@ -133,9 +138,14 @@ spec:
             secretName: {{ include "server.fullname" . }}-encryption-key
 {{- end }}
 {{- if .Values.config.admin.enabled }}
-        - name: initial-admin-secret
+        - name: initial-admin-password-secret
           secret:
             secretName: {{ include "server.fullname" . }}-initial-admin-secret
+{{- if .Values.config.admin.accessKeySecret }}
```

Comment:
What are the steps for a user should securely generate this? And create the secret?

Reply:
```
kubectl create secret generic my-access-key --from-literal=access-key=$(openssl rand -hex 5).$(openssl rand -hex 12)
```

Reply:
thanks!